Containerfile

```diff
@@ -1,54 +1,38 @@
 FROM ubuntu
 
-LABEL maintainer="vchalla@redhat.com"
-
+# setup installation directory and disable prompts during installation
 WORKDIR /tmp
 ARG DEBIAN_FRONTEND=noninteractive
 
-# ALPINE
+# Install necessary libraries for subsequent commands
 RUN apt-get update
 RUN apt-get upgrade  -y
-RUN apt-get install -y podman
-RUN apt-get install -y wget python3.6 git dumb-init
-
-# Snafu Build Dependencies
-# TODO: A lot of the *-dev dependencies were added while attempting to build
-#       snafu's requirements (mainly numpy, scipy, pandas). Now that these are
-#       installed using Alpine's package manager, they may not all be necessary
-#       anymore.
-RUN apt-get install -y python3-numpy python3-scipy python3-pandas gcc python3-dev \
-                       postgresql-client libffi-dev libxml2 \
-                       libxml2-dev libxslt-dev libjpeg-dev \
-                       zlib1g-dev musl-dev
+RUN apt-get install -y podman wget git dumb-init python3.6 python3-distutils python3-pip python3-apt
 
-# RUN apt-get install -y libressl-dev
-# Install required third-party packages
+# Install vegeta for HTTP benchmarking
 RUN wget https://github.com/tsenart/vegeta/releases/download/v12.8.3/vegeta-12.8.3-linux-amd64.tar.gz
 RUN tar -xzf vegeta-12.8.3-linux-amd64.tar.gz
 RUN mv vegeta /usr/local/bin/vegeta
```

Comment:
All these need to happen in one step, and the tarball needs to be deleted.

Reply:
Changed as suggested. Thanks!